src/org/zaproxy/zap/extension/advreport/AlertsPanel.java

```diff
@@ -36,51 +44,98 @@ private void initialize( List<String> alertTypes ){
 		gbc.gridx = 0;
 		gbc.gridy = 0;
 		gbc.fill = GridBagConstraints.HORIZONTAL;
-		
+		gbc.insets = new Insets(2,5,30,10);
+
+		selectionPanel.add(getRiskBox("High"), gbc);
+
+		gbc.gridx++;
+		selectionPanel.add(getRiskBox("Medium"), gbc);
+
+		gbc.gridx++;
+		selectionPanel.add(getRiskBox("Low"), gbc);
+
+		gbc.gridx++;
+		selectionPanel.add(getRiskBox("Informational"), gbc);
+
+		gbc.insets = new Insets(0,20,2,0);
+		gbc.gridy = 1;
+		gbc.gridwidth = 4;
+
 		for( String alertType: alertTypes ){
-			JLabel label = new JLabel(alertType);
-		    JCheckBox selection = new JCheckBox();
-		    selection.setSelected( true );
-		    selection.setName( alertType );
-		    
-		    gbc.gridx = 0;
-		    gbc.anchor = GridBagConstraints.WEST;
-		    selectionPanel.add(label, gbc);
-			
-		    gbc.gridx = 1;
-		    gbc.anchor = GridBagConstraints.EAST;
-		    selectionPanel.add(selection, gbc);
-		    this.getSelections().add( selection );	
-			
+
+			JCheckBox selection = new JCheckBox();
+			selection.setText(alertType);
+			selection.setSelected( true );
+			selection.setName( alertType );
+
+			gbc.gridx = 0;
+			gbc.anchor = GridBagConstraints.CENTER;
+			selectionPanel.add(selection, gbc);
+			this.getSelections().add( selection );
 			gbc.gridy += 1;
+
 		}
-		 				

```

Comment:
(Unrelated) Indentation changes should not be done in the same commit as actual code changes. [1]
It adds a lot of noise to the changes and makes review process cumbersome.

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines